.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Test using pytest
```

Comment:
I already added this workflow in this PR #11. It is still marked as draft because I'm waiting to fix the container registry permissions for one of the workflows

Reply:
Should I delete mine or combine my contents with yours?